components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -1983,20 +1986,29 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Comment'
+        304:
+          description: |
+            Not Modified.
+            Empty body because the client has already the latest version of the requested resource.
+          content: {}
```

Comment:
remove if not necessary

Reply:
The 'Not Modified' response has been added when the user sends an edit request without modifying anything. Should I remove it?